File: openapi-generator/dart-jaguar/lib/model/asset.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:openapi/model/asset_cable.dart';
part 'asset.jser.dart';

class Asset {
   /* Serial number of the host asset */
  @Alias('assetSerialNumber')
  final String assetSerialNumber;
   /* The cable connected to the asset */
  @Alias('cable')
  final List<AssetCable> cable;
   /* Engine hours */
  @Alias('engineHours')
  final int engineHours;
   /* Asset ID */
  @Alias('id')
  final int id;
   /* Asset name */
  @Alias('name')
  final String name;
  

  Asset(
      

{
     this.assetSerialNumber = null,  
     this.cable = const [],  
     this.engineHours = null,  
    
     this.id = null,   this.name = null 
    
    }
  );

  @override
  String toString() {
    return 'Asset[assetSerialNumber=$assetSerialNumber, cable=$cable, engineHours=$engineHours, id=$id, name=$name, ]';
  }
}

@GenSerializer(nullableFields: true)
class AssetSerializer extends Serializer<Asset> with _$AssetSerializer {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/Asset.scala

package org.openapitools.client.model


case class Asset (
    /* Serial number of the host asset */
    _assetSerialNumber: Option[String],
    /* The cable connected to the asset */
    _cable: Option[List[AssetCable]],
    /* Engine hours */
    _engineHours: Option[Integer],
    /* Asset ID */
    _id: Long,
    /* Asset name */
    _name: Option[String]
)
object Asset {
    def toStringBody(var_assetSerialNumber: Object, var_cable: Object, var_engineHours: Object, var_id: Object, var_name: Object) =
        s"""
        | {
        | "assetSerialNumber":$var_assetSerialNumber,"cable":$var_cable,"engineHours":$var_engineHours,"id":$var_id,"name":$var_name
        | }
        """.stripMargin
}
